.github/actions/dispatch-and-download/action.yml

```diff
@@ -27,8 +27,9 @@ runs:
   using: "composite"
   steps:
     - name: Log inputs to job summary
+      shell: bash
```

Comment:
🤔 It normally executes in `zsh` and that causes an issue with the steps? Or just being explicit to be safe and consistent with other steps?

Reply:
Actions require a shell to be specified either inside of it or by the caller. Went with defining this inside to reduce usage friction and ensure that we don't get weird errors due to different shells. 

No good reason for bash vs other options tho, just my (ignorant) default.

Reply:
To be clear, I'm 100% fine with bash 😄 just didn't know it needed to be specified. Cool beans! 👍🏻